index.html

```diff
@@ -0,0 +1,132 @@
+    <link rel="stylesheet " href="/css/main.css" />
+    <link rel="stylesheet " href="/css/footer.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <h1>
+          <a id="logo" href="/" rel="home" aria-label="판다마켓 홈"> </a>
+        </h1>
+        <a href="/login.html" id="login-button">로그인</a>
+      </div>
+    </header>
+    <main>
+      <section class="top-bottom-section part-1">
+        <div>
+          <h2 class="sub-title">
+            일상의 모든 물건을 <br />
+            거래해 보세요
+          </h2>
+          <a href="/items.html">구경하러 가기</a>
+        </div>
+        <div>
+          <img src="imgs/Img_home_top.png" alt="언덕위 판다" />
+        </div>
+      </section>
+      <section class="main-section part-2">
+        <div>
+          <img src="imgs/Img_home_01.png" alt="판다 이미지 1" />
+        </div>
+        <div>
+          <div class="highlight">Hot itme</div>
+          <h2 class="sub-title">
+            인기 상품을 <br />
+            확인해 보세요
+          </h2>
+          <p class="description">
+            가장 HOT한 중고거래 물품을 <br />
+            판다 마켓에서 확인해 보세요
+          </p>
+        </div>
+      </section>
+      <section class="main-section part-3">
+        <div>
+          <div class="highlight">Search</div>
+          <h2 class="sub-title">
+            구매를 원하는<br />
+            상품을 검색하세요
+          </h2>
+          <p class="description">
+            구매하고 싶은 물품은 검색해서 <br />
+            쉽게 찾아보세요
+          </p>
+        </div>
+        <div>
+          <img src="imgs/Img_home_02.png" alt="판다 이미지 2" />
+        </div>
+      </section>
+      <section class="main-section part-4">
+        <div>
+          <img src="imgs/Img_home_03.png" alt="판다 이미지 3" />
+        </div>
+        <div>
+          <div class="highlight">Register</div>
+          <h2 class="sub-title">
+            판매를 원하는 <br />
+            상품을 등록하세요
+          </h2>
+          <p class="description">
+            어떤 물건이든 판매하고 싶은 상품을<br />
+            쉽게 등록하세요
+          </p>
+        </div>
+      </section>
+      <section class="top-bottom-section part-5">
+        <div>
+          <h2 class="sub-title">
+            일상의 모든 물건을 <br />
+            거래해 보세요
+          </h2>
+        </div>
+        <div>
+          <img src="imgs/Img_home_bottom.png" alt="언덕위 판다" />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
+        <small class="footer-copy">© codeit · 2024</small>
+        <nav class="footer-nav" aria-label="푸터 내비게이션">
+          <ul class="footer-links" role="list">
+            <li><a href="/privacy.html">Privacy Policy</a></li>
+            <li><a href="/faq.html">FAQ</a></li>
+          </ul>
+        </nav>
+        <ul class="footer-links" role="list">
```

Comment:
❗️ 수정요청
ul 태그는 이미 리스트 태그이므로 role 속성을 통해 list로 명시해주실 필요가 없습니다.

```suggestion
        <ul class="footer-links">
```